packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -333,7 +332,6 @@ contract Deploy is Deployer {
         console.log("Deploying OP Chain");
 
         // Ensure that the requisite contracts are deployed
-        address superchainConfigProxy = artifacts.mustGetAddress("SuperchainConfigProxy");
```

Comment:
why don't we need this?

Reply:
It was only used to feed into the DelayedWETH initialization call